OsmAnd/res/values/strings.xml

```diff
@@ -1767,24 +1766,24 @@
     <string name="enable_sleep_mode">Enable GPS background mode</string>
     <string name="save_track_to_gpx_globally">Log track to GPX file</string>
     <string name="save_track_to_gpx_globally_headline">On demand track logging</string>
-    <string name="save_track_to_gpx_globally_descr">General position logging to a GPX file can be turned on or off using the GPX logging widget on the map screen.</string>
+    <string name="save_track_to_gpx_globally_descr">General position logging to a GPX file can be turned on or off using the GPX logging widget on the map.</string>
     <string name="save_current_track_descr">Save current track as GPX file now.</string>
     <string name="save_current_track">Save current track</string>
     <string name="save_track_to_gpx">Auto-record track during navigation</string>
     <string name="save_track_to_gpx_descrp">A GPX track is automatically saved to the tracks folder during navigation.</string>
     <string name="save_track_interval_globally">Logging interval</string>
     <string name="save_track_interval">Logging interval during navigation</string>
-    <string name="save_track_interval_descr">Choose logging interval for track recording during navigation.</string>
+    <string name="save_track_interval_descr">Choose logging interval for track recording during navigation</string>
     <string name="voice_provider_descr">Select the voice guidance for navigation.</string>
     <string name="voice_provider">Voice guidance</string>
     <string name="enable_proxy_title">Enable HTTP proxy</string>
-    <string name="enable_proxy_descr">Configure an HTTP proxy for all network requests.</string>
+    <string name="enable_proxy_descr">Set up a HTTP proxy for all network requests.</string>
```

Comment:
Hello! 
Could you please tell, whey "Set up" is better than "Configure"?

Reply:
Yes, I also do not agree with the suggested modifications of replacing "Configure" by "Set up". Will remove for now, unless where it is clearly ok.

Reply:
The idea was to arrive at some consistency. Configure/configuration, vs set up/setup. The latter is marginally easier to understand, I think easier to translate, and a bit shorter.